lib/cli.js

```diff
@@ -48,17 +50,24 @@ e.g. --no-reload rather than --reload=false}
   flagOptions
 );
 
-const argv = Object.assign({}, cli.flags);
-const explorer = cosmiconfig('serve', {});
 // webpack-serve config from serve.config.js or supported format
-const { config: serveConfig = {} } = explorer.searchSync() || {};
-let config;
+const searchPlaces = [
```

Comment:
Is this a requirement now ? :)  Looks like the old defaults (`cosmiconfig =< v4.0.0`) I'm aware of this change made in `v5.0.0` but still tought this was **optional** and still included some `searchPlaces` by default (maybe I'm wrong though)

Reply:
I don't know, I copied that from the master branch, I'll check it later, I thought if we want to provide custom search places, we should provide all the possibilities, but I will check that